src/util/Options.js

```diff
@@ -0,0 +1,48 @@
+import { BiUser, BiHistory, BiMoney, BiDish, BiStar } from "react-icons/bi";
+import { GiKnifeFork } from "react-icons/gi";
+
+export const userServiceOptions = [
+  {
+    name: "Perfil",
+    route: "/perfil",
+    icon: BiUser,
+  },
```

Comment:
Ao invés de ter dois arrays, seria melhor ter um só com todas as rotas (e não repetir nenhuma, a de `/cardapio` pode ser renomeada, etc), já que todas precisam ser renderizadas no Router. Além disso, pra facilitar mais informações poderiam estar presentes.

```suggestion
  {
    name: "Perfil",
    route: "/perfil",
    icon: BiUser,
    component: <Home />,
    type: UserRoute
  },
```